src/chocolatey.install/chocolatey.en-us.wxl

```diff
@@ -4,6 +4,7 @@
     <String Id="NewerVersionInstalled" Overridable="yes">A newer version of Chocolatey is already installed.</String>
     <String Id="ProfileAdditionDescription" Overridable="yes">Add the Chocolatey Helpers to your PowerShell Profile to enable tab-completions.
 Defaults to adding to the AllUsersAllHosts profile.</String>
-    <String Id="RequiresDotNet48" Overridable="yes">This application requires .NET Framework 4.8. Please install the .NET Framework then run this installer again.</String>
-    <String Id="ToolsFolderDescription" Overridable="yes">Sets the location for the Tools Folder</String>
-</WixLocalization>
\ No newline at end of file
+    <String Id="RequiresDotNet48" Overridable="yes">Chocolatey CLI application requires .NET Framework 4.8. Please install the .NET Framework then run this installer again.</String>
```

Comment:
The wording seems a little off here, shouldn't this be:
```suggestion
    <String Id="RequiresDotNet48" Overridable="yes">Chocolatey CLI requires .NET Framework 4.8. Please install the .NET Framework then run this installer again.</String>
```

Reply:
Good catch. I had that [above](https://github.com/chocolatey/choco/pull/3290#discussion_r1281648302) but didn't notice it had been missed.

Reply:
Apologies, fi thought I had copy/pasted the message from the suggestion, but it seems I didn't. I've updated this now.